src/Nethereum.Hex/HexTypes/HexBigInteger.cs

```diff
@@ -24,5 +24,10 @@ public override bool Equals(object obj)
 
             return false;
         }
+
+        public override string ToString()
+        {
+            return value.ToString();
```

Comment:
value should be Value, as it is lazy loaded (Hopefully it makes sense) https://github.com/Nethereum/Nethereum/blob/7dba906e3dfb0cb01c5e2d9acb49b5df1fc15ffa/src/Nethereum.Hex/HexTypes/HexType.cs#L55